workbench/header.php

```diff
@@ -102,8 +102,12 @@ function strip_seps($haystack) {
         rsort($betaTagNames);
         rsort($gaTagNames);
 
-        $latestBetaVersion = strip_seps($betaTagNames[0]);
-        $latestGaVersion = strip_seps($gaTagNames[0]);
+        if (count($betaTagNames) > 0) {
```

Comment:
when I was debugging errors for HTTPs this one ended up sending errors since array was empty